assets/components/src/plugin-installer/index.js

```diff
@@ -167,6 +165,12 @@ class PluginInstaller extends Component {
 		}
 		return (
 			<div>
+				{ ! pluginInfo.length && (
```

Comment:
The spinner is always active in my testing, even after loading is complete:
![localhost_wp-admin_admin php_page=newspack-subscriptions-wizard (2)](https://user-images.githubusercontent.com/7317227/60441977-ecccf700-9bcc-11e9-874b-fa8db7f35510.png)

This is because once plugin info is retrieved, the `pluginInfo` field is an object, not an array (so `length` is `undefined`, and `! length` is always `true`).

Reply:
Good catch! This should fix it up: https://github.com/Automattic/newspack-plugin/pull/140/commits/6ea3d64d59a584df6f7ea8992aab11a1be90ab22